src/docfx/lib/schema/JsonSchemaValidator.cs

```diff
@@ -171,11 +171,16 @@ private void ValidateString(JsonSchema schema, string name, JValue scalar, strin
                     errors.Add(Errors.StringLengthInvalid(JsonUtility.GetSourceInfo(scalar), scalar.Path, $">= {schema.MinLength}"));
             }
 
+            if (schema.Pattern != null && !Regex.IsMatch(str, schema.Pattern))
```

Comment:
>schema.Pattern [](start = 62, length = 14)

what if schema.Pattern is invalid e.g. '[' ?

Reply:
We'll just throw whatever `Regex.IsMatch` throws. Currently we consider template related errors system errors.